### ISSUE TITLE
Failure to upload for no reason

### ISSUE THREAD

**Issue by juliathemew64** [developersu/ns-usbloader-mobile#11]
Hello just am trying to send a nsp file using the program to my switch. Tried using both tinfoil and goldleaf with no change in result. I go to upload the file and it loads then nothing happens. Then I press the button again and I just get a upload failure. Pretty major bug. Hope it gets fixed soon as there's alot of games above 4 gbs

**Comment by developersu** [comment]
Thank you for reporting.
Hmm.. it's strange. App should be asking for permissions or something.. I guess. Does your phone recognizes connected switch? Like it shows pop-up screen or something?

**Comment by juliathemew64** [comment]
J

**Comment by juliathemew64** [comment]
Yeah I got the permission prompt and gold lead shows the usb icon. I just downgraded goldleaf to version 0.5 and im still having the same issue

**Comment by juliathemew64** [comment]
Forgot to mention im using a Samsung note 9

**Comment by developersu** [comment]
Will find someone with Samsung and test. No idea what's wrong here ><

**Comment by developersu** [comment]
Should be fixed on latest version. If not, please open new issue. 
I don't have samsung. Tested on OnePlus and Oppo
